example/README.md

```diff
@@ -1,88 +1,223 @@
 ObjectBox Examples
 ==========================
 
-Flutter
---------
-* See a [Flutter example app](flutter/objectbox_demo) for code examples - requires Flutter 1.12
+If you want to dive straight into the code:
 
-As opposed to a plain Dart app which runs directly on your PC, there are more restrictions where your Flutter app can 
-write data. Therefore, you should give ObjectBox a full path to a per-app documents directory, where to store the data 
-even when a user closes your app. 
+* see a [Flutter example app](flutter/objectbox_demo)
 
-If you didn't specify this path to ObjectBox, it would try to use a default "objectbox" directory where the app is 
-currently running, but it doesn't have permissions to write there: `failed to create store: 10199 Dir does not exist: objectbox (30)`.
+Data model
+--------------------
+
+In the following file, e.g. `models.dart`, we import ObjectBox to access annotations.
+Then, we define a single "Entity" that should be persisted by ObjectBox. You can have multiple
+entities in the same file, or you can have them spread across multiple files in your package's `lib` directory.
+
+```dart
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class Note {
+    // Each "Entity" needs a unique integer ID property.
+    // Add `Id()` annotation if its name isn't "id" (case insensitive).
+    int id;
+
+    String text;
+
+    int date;
+
+    @Transient() // Make this field ignored, not stored in the database.
+    int notPersisted;
+
+    // An empty default constructor is needed but you can use optional args.
+    Note({this.text});
+
+    // Note: just for logs in the examples below(), not needed by ObjectBox.
+    toString() => 'Note{id: $id, text: $text}';
+}
+```
+
+To generate ObjectBox binding code for your entities, run `(flutter|dart) pub build_runner build`.
```

Comment:
`(flutter|dart) pub build_runner build` for a novice, isn't this confusing? (Or does this or syntax actually work?)

Reply:
```suggestion
To generate ObjectBox binding code for your entities, run `pub run build_runner build`.
```